source/s3_auto_ranged_get.c

```diff
@@ -397,18 +396,24 @@ static int s_s3_auto_ranged_get_prepare_request(
     aws_s3_request_setup_send_data(request, message);
     aws_http_message_release(message);
 
+    /* Success! */
     AWS_LOGF_DEBUG(
         AWS_LS_S3_META_REQUEST,
         "id=%p: Created request %p for part %d",
         (void *)meta_request,
         (void *)request,
         request->part_number);
 
-    return AWS_OP_SUCCESS;
+    success = true;
 
-message_alloc_failed:
-
-    return AWS_OP_ERR;
+finish:;
```

Comment:
nit: extra ;

Reply:
it's actually necessary, because I declare a variable on the next line.
See: https://stackoverflow.com/questions/8384388/variable-declaration-after-goto-label